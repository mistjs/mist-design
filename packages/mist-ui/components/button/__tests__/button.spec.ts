import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import Button from '../button';

describe('button', () => {
  it('renders correctly', () => {
    const wrapper = mount(Button, { slots: { default: () => 'Test' } });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
