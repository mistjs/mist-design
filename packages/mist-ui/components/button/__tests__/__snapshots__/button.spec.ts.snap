// Vitest Snapshot v1

exports[`button > render properly 1`] = `"<button>Test</button>"`;

exports[`button > renders correctly 1`] = `"<button>Test</button>"`;
